packages/amplify-category-geo/src/service-utils/mapUtils.ts

```diff
@@ -51,7 +61,8 @@ export const modifyMapResource = async (context: $TSContext, parameters: MapPara
   const mapStack = new MapStack(new App(), 'MapStack', { ...parameters, ...templateMappings, authResourceName });
   generateTemplateFile(mapStack, parameters.name);
   saveCFNParameters(parameters);
-  stateManager.setResourceInputsJson(pathManager.findProjectRoot(), category, parameters.name, { groupPermissions: parameters.groupPermissions });
+  stateManager.setResourceInputsJson(pathManager.findProjectRoot(),
+    category, parameters.name, { groupPermissions: parameters.groupPermissions });
```

Comment:
Same suggestion here, but the surrounding code looks so similar I wonder if it should be pulled into a function

```suggestion
  stateManager.setResourceInputsJson(
    undefined,
    category,
    parameters.name,
    { groupPermissions: parameters.groupPermissions },
  );
```